reconcile/queries.py

```diff
@@ -22,6 +22,48 @@ def get_app_interface_settings():
     return None
 
 
+APP_INTERFACE_EMAILS_QUERY = """
+{
+  emails: app_interface_emails_v1 {
+    name
+    subject
+    to {
+      aliases
+      services {
+        serviceOwners {
+          email
+        }
+      }
+      clusters {
+        name
+      }
+      namespaces {
+        name
+      }
+      aws_accounts {
+        name
+      }
+      roles {
+        users {
+          org_username
+        }
+      }
+      users {
+        org_username
+      }
+    }
+    body
+  }
+}
+"""
+
+
+def get_app_interface_emails():
+    """ Returns Email resources defined in app-interface """
+    gqlapi = gql.get_api()
+    return gqlapi.query(APP_INTERFACE_EMAILS_QUERY)['emails']
```

Comment:
It's probably good... app-interface counterpart still not ready, so I'm not able to verify this.

Reply:
It's good! :)